include/slang-rhi.h

```diff
@@ -294,6 +294,7 @@ struct FormatInfo
     bool isSigned : 1;
     bool isSrgb : 1;
     bool isCompressed : 1;
+    bool supportsNonePowerOf2 : 1;
```

Comment:
shouldn't that be spelled `supportsNonPowerOf2` ? In addition I would probably prefer `supportsNonPowerOfTwo`